custom/icds_reports/reports/bihar_api.py

```diff
@@ -68,3 +68,71 @@ def get_api_demographics_data(month, state_id, last_person_case_id):
     # To apply pagination on database query with data size length
     limited_demographics_data = list(demographics_data_query[:CAS_API_PAGE_SIZE])
     return limited_demographics_data,  get_total_records_count(month, state_id)
+
+
+def get_vaccine_total_records_count(month, state_id):
```

Comment:
can you just cache this function as this count will always be same

Reply:
done